client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -72,6 +72,7 @@ const PluginsBrowserListElement = ( props ) => {
 			site: site,
 			plugin: plugin.slug,
 			list_name: props.listName,
+			grid_position: props.gridPosition,
```

Comment:
adding `blog_id` here would be helpful in the future, something like 
```suggestion
			grid_position: props.gridPosition,
			blog_id: selectedSite?.ID,
```

Reply:
Added at https://github.com/Automattic/wp-calypso/pull/62731/commits/0b1dbd5f3b9f347660e8a5f12a145eb0c2e8de15